### PR TITLE
End to end cleanup of windows binary building

### PR DESCRIPTION
**Pull request by mwrock**

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/hab/win/win-build.ps1 [hunk 1]
```diff
@@ -58,7 +58,7 @@ if (-not (Test-AppVeyor)) {
     # We need the Visual C++ tools to build Rust crates (provides a compiler and linker) 
     choco install 'visualcppbuildtools' --version '14.0.25123' --confirm --allowemptychecksum
 
-    choco install 7zip
+    choco install 7zip --version '16.02.0.20160811' --confirm
```
Comment (rrxtns) [inline components/hab/win/win-build.ps1]: Is there a strong need to pin this version?

**Comment by reset** [comment]
gif-keyboard-16385737110447825001

**Comment by reset** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/167513614).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/167513614).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**